packages/apps/src/initSettings.ts

```diff
@@ -24,6 +24,20 @@ function getApiUrl (): string {
 
     assert(url.startsWith('ws://') || url.startsWith('wss://'), 'Non-prefixed ws/wss url');
 
+    const urlArray = ['127.0.0.1', 'localhost', '.cere.network'];
+    let hostname = (new URL(url)).hostname;
+    const port = Number(new URL(url).port);
```

Comment:
Create the URL object once and use it everywhere.

Reply:
done.